src/shared/organisms/DataPanel/DataPanel.tsx

```diff
@@ -512,8 +512,7 @@ const DataPanel: React.FC<Props> = ({}) => {
         const metadataFiles = value.filter(
           v =>
             v?.localStorageType === 'resource' &&
-            v['@type'] !== 'File' &&
-            v['@type'].includes('File')
+            (v['@type'] !== 'File' || v['@type'].includes('File'))
```

Comment:
Shouldn't we check that the `File` is *not* included in the type array (i.e. `!v['@type'].includes('File')`)?